core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/CompletionGroup.java

```diff
@@ -0,0 +1,41 @@
+package uk.ac.bbsrc.tgac.miso.core.data;
+
+import java.util.TreeMap;
+
+import uk.ac.bbsrc.tgac.miso.core.data.type.HealthType;
+
+public class CompletionGroup extends TreeMap<HealthType, PoolOrderCompletion> {
```

Comment:
Consider renaming to PoolOrderCompletionGroup to match PoolOrderCompletion and avoid ambiguity?